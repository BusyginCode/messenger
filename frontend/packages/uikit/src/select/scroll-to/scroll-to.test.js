import React from 'react'
import { shallow } from 'enzyme'
import ScrollTo from './scroll-to'

describe('uikit/Select/ScrollTo', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ScrollTo />)
    expect(wrapper).toMatchSnapshot()
  })
})
