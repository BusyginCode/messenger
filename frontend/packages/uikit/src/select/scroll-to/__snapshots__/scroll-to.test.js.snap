// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/Select/ScrollTo renders correctly 1`] = `<div />`;
